app/scripts/metamask-controller.js

```diff
@@ -1302,13 +1302,12 @@ export default class MetamaskController extends EventEmitter {
     this.preferencesController.setAddresses(newAccounts);
     newAccounts.forEach((address) => {
       if (!oldAccounts.includes(address)) {
+        const pathDescription = hdPath === `m/44'/60'/0'` ? ' (legacy)' : '';
```

Comment:
Hmmm.... we should localize this string :thinking:  How do we get a localization here though? Those are in the UI.

Maybe we can pass in a `hdPathDescription` as an additional parameter? :shrug: 

That might work out well, since the list of supported HD paths is in the UI as well. So when we add more paths, we can add descriptions for them there as well.

Reply:
Done!